Description: Add missing file contain translation to the POTFILES.in file
 Put string "uglib/UgPlugin-aria2.c" in po/POTFILES.in

Forwarded: not-needed
Author: Elías Alejandro Año Mendoza <ealmdz@gmail.com>
Last-Update: 2011-06-09

--- uget.orig/po/POTFILES.in	2011-06-09 17:35:17.000000000 -0500
+++ uget/po/POTFILES.in	2011-06-09 17:36:36.000000000 -0500
@@ -9,6 +9,7 @@
 uglib/UgMessage.c
 uglib/UgOption.c
 uglib/UgPlugin.c
+uglib/UgPlugin-aria2.c
 uglib/UgPlugin-curl.c
 uglib/UgRegistry.c
 uglib/UgRunning.c
